flowpm/tfpm.py

```diff
@@ -253,16 +249,15 @@ def drift(state, ai, ac, af, cosmology=Planck15, dtype=np.float32,
   with tf.name_scope(name):
     state = tf.convert_to_tensor(state, name="state")
 
-    pt = PerturbationGrowth(cosmology, a=[ai, ac, af], a_normalize=1.0)
-    fac = 1. / (ac ** 3 * pt.E(ac)) * (pt.Gp(af) - pt.Gp(ai)) / pt.gp(ac)
+    fac = 1. / (ac ** 3 * E(cosmo,ac)) * (D1(cosmo,af) - D1(cosmo,ai)) / D1f(cosmo,ac)
```

Comment:
ok great, but does that mean we don't need the Gp and gp functions in `background.py`?